auth_admin_passkey/model/res_users.py

```diff
@@ -132,6 +134,13 @@ def check_credentials(self, cr, uid, password):
                     cr, uid, password)
                 return True
             except exceptions.AccessDenied:
-                return self.check_credentials(cr, SUPERUSER_ID, password)
+                icp_obj = self.pool['ir.config_parameter']
+                allow_db_pwd = safe_eval(icp_obj.get_param(
+                    cr, SUPERUSER_ID, 'auth_admin_passkey.db_pwd', 'False'))
+                if allow_db_pwd and password and\
+                        password == config['db_password']:
+                    return True
```

Comment:
could you extract these 6 lines to a separate method so that the behavior can be customized?